changelog.d/371.misc

```diff
@@ -0,0 +1 @@
+Update return types on converted async functions.
```

Comment:
I consolidate the changelogs in 48eaeb237c8463151b16674edcc17e8de752c055, so let's be consistent:

```suggestion
Add type hints and validate with mypy.
```